tests/regression/run-api-with-db.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+source tests/regression/common.sh
+
+check_env
+setup_build_opts
+
+# Get command line arguments
+# Current arguments parsed are <socket name> <remaining_args>
+get_cli_args $@
+
+# Install prerequisites
+make
+pip3 install --user -r tests/regression/requirements.txt
```

Comment:
Does it make sense to install Python packages right after `make`? How about moving the required packages into another script?

Reply:
buildkite would use the downloaded packages in cache, so We somehow not need to move the installing command away from this script